src/graphics/program-lib/chunks/blendNormals.frag

```diff
@@ -0,0 +1,22 @@
+// https://blog.selfshadow.com/publications/blending-in-detail/#detail-oriented
+vec3 blendNormals_rnm(vec3 n1, vec3 n2) {
+    n1 += vec3(0, 0, 1);
+    n2 *= vec3(-1, -1, 1);
+    return n1*dot(n1, n2)/n1.z - n2;
```

Comment:
should this not be normalized like other blends? Even linked web page normalizes the result of rnm blend.

Reply:
even though from comments on that link it seems that if both input normals are normalized, the output doesn't need to be.

Reply:
Might be related, but we do not normalize normals intentionally, in order to have Toksvig AA on specularity: https://github.com/playcanvas/engine/issues/1818

Reply:
looking at the history of normalMap.frag, we normalize them now since Jan 2020.

Reply:
@mvaligursky Fixed